requirements.txt

```diff
@@ -1,18 +1,18 @@
-Click==7.0
 Flask==1.1.1
 Jinja2==2.11.1
 MarkupSafe==1.1.1
 Werkzeug==1.0.0
 attrs==19.1.0
 certifi==2019.11.28
 chardet==3.0.4
-idna==2.8
-importlib-metadata==1.5.0
+click==7.1.1
+idna==2.9
+importlib-metadata==1.5.1
 itsdangerous==1.1.0
 jsonschema==3.2.0
-pyrsistent==0.15.7
-requests==2.22.0
+pyrsistent==0.16.0
+requests==2.23.0
 six==1.14.0
 strict-rfc3339==0.7
 urllib3==1.25.8
-zipp==2.2.0
+zipp==3.1.0
```

Comment:
This one seems like a big change. No breaking changes? Perhaps run all the unit tests to verify? Or undo this change?

Reply:
oof.. this must have been updated in a make setup. I will remove this file from the check in